go/tools/release-notes/release_notes.go

```diff
@@ -495,7 +495,7 @@ func main() {
 	flag.Parse()
 
 	// The -version flag must be of a valid format.
-	rx := regexp.MustCompile(`v(\d+)\.(\d+)\.(\d+)`)
+	rx := regexp.MustCompile(`v([0-9]+)\.([0-9]+)\.([0-9]+)`)
```

Comment:
why did we need this?

Reply:
It does basically the same thing and the release branches are already using `[0-9]` instead of `\d`.